### ISSUE TITLE
export es5 instead of es6

### ISSUE THREAD

**Issue by rickyp-uber** [snowcoders/react-popover#1]
I'll try and create a reproduction environment so we can test this properly but it seems that the es6 isn't supported by node.js 6.10.0 (http://node.green/) which is going to limit usage of this library. This means that server side rendering for this component doesn't work.

We've tried a few things (getting babel to translate the node_module) but can't figure out a way to do it without putting some babel keywords into this library.

Basically, mind putting this at es5 and commonjs for the short term and I can work with you to figure out what a longer term solution is because I do agree, we should just be outputing es6 modules at this point.

**Comment by k2snowman69** [comment]
6.10.0 is over a year old... can you not upgrade your node.js server?

I'm okay taking this (actually ran into similar issues with jsdom and other server side test frameworks) for now but expect in 6 months or so for us to move back to es6.
